src/Microsoft.PowerApps.TestEngine/TestInfra/PlaywrightTestInfraFunctions.cs

```diff
@@ -205,8 +205,7 @@ public async Task GoToUrlAsync(string url)
                 Page = await BrowserContext.NewPageAsync();
             }
 
-            // TODO: consider whether to make waiting for network idle state part of the function input
-            var response = await Page.GotoAsync(url, new PageGotoOptions() { WaitUntil = WaitUntilState.NetworkIdle });
+            var response = await Page.GotoAsync(url);
```

Comment:
What is the default "WaitUntil" setting? I tried it with "WaitUntilState.Load" which seemed to work perfectly.

Reply:
I think I answered my own question. Looks like the [default is "Load"](https://github.com/microsoft/playwright/blob/5821c547aacfd542259c8a9707b1eee119f0b692/packages/playwright-core/src/client/page.ts#L409)

Reply:
Correct. The `WaitUntil` property defaults to `Load` even in the PageGotoOptions object.